src/meta/processors/partsMan/ListPartsProcessor.cpp

```diff
@@ -0,0 +1,111 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "meta/ActiveHostsMan.h"
+#include "meta/processors/partsMan/ListPartsProcessor.h"
+#include "meta/processors/admin/AdminClient.h"
+
+namespace nebula {
+namespace meta {
+
+void ListPartsProcessor::process(const cpp2::ListPartsReq& req) {
```

Comment:
Do you think we can support list the partition specified in console? If we have to many partitions, it's hard to look for  a single partition.

Reply:
Agreed.  We'd better support to show some part  information specified too.

Reply:
The result is sorted by partition ID.

Reply:
Of course,  we could open another pr for it.

Reply:
Yep, would it be better to use the `LIMIT` clause?